CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 - Update api version in ocp templates for image, buildconfig, route and deploymentconfig ([#1072](https://github.com/opendevstack/ods-jenkins-shared-library/issues/1072))
 - SonarQube Maintenance (LTS update, DB update, SAML setup) ([#1211](https://github.com/opendevstack/ods-core/issues/1211))
-
+- Jenkins Updgrade to v2.426.3 [#1270](https://github.com/opendevstack/ods-core/pull/1270)
```

Comment:
Typo - update or upgrade ?

Actually this change includes additional updates.

Reply:
You are right @tbugfinder, we have updated the Changelog entry